### DISCUSSION TITLE
A new place to grow

### DISCUSSION THREAD

**Discussion by jumoog** [intro-skipper/intro-skipper#296] [Announcements]
What started as a private fork to fix small bugs has grown so much that it is time to move the work into its own organisation. This is the right place to make the plugin even better! I would like to thank all the people who have [contributed](https://github.com/intro-skipper/intro-skipper/graphs/contributors) to the development.

A special thanks go to @AbandonedCart and @rlauuzo! 🍻

**Comment by AbandonedCart** [comment]
Aww, you’re so sweet.
